bats/tests/profile/create-profile-output.bats

```diff
@@ -0,0 +1,284 @@
+load '../helpers/load'
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start app' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+@test 'report invalid parameters for plist' {
+    run rdctl create-profile --output=plist --from-settings --hive=fish
+    assert_failure
+    assert_output --partial "registry hive and type can't be specified with plist"
+}
+
+@test 'report unrecognized output-options' {
+    run rdctl create-profile '--output=pickle'
```

Comment:
Why did you quote the last argument? It is not wrong, but seems like a curious choice.

Reply:
I don't recall, but it is one of the remaining unnecessary quoted words in the file.